webapp/packages/core-blocks/src/Tree/TreeNode/TreeNodeControl.m.css

```diff
@@ -0,0 +1,41 @@
+.treeNodeControl {
+  composes: theme-ripple theme-ripple-selectable from global;
+  box-sizing: border-box;
+  height: 22px;
+  display: flex;
+  align-items: center;
+  padding: 2px 0 2px 0;
+  user-select: none;
+  white-space: nowrap;
+  position: initial !important;
+  outline: none;
+
+  &::before {
+    left: 0 !important;
+    top: auto !important;
+    height: inherit !important;
+    width: 100% !important;
+  }
+
+  &.dragging {
+    opacity: 0.6;
+  }
```

Comment:
please move this style to nav-tree components and remove `dragging` and `editing` props from base `TreeNodeControl`